app/validators/questions/mutually_exclusive_validator.py

```diff
@@ -6,14 +6,19 @@ class MutuallyExclusiveQuestionValidator(QuestionValidator):
     MUTUALLY_EXCLUSIVE_CONTAINS_MANDATORY = (
         "MutuallyExclusive question type cannot contain mandatory answers."
     )
-    NON_CHECKBOX_ANSWER = "Question is not of type Checkbox."
+    NON_CHECKBOX_RADIO_ANSWER = "Question is not of type Checkbox or Radio."
 
     def validate(self):
         super().validate()
 
         if any(answer["mandatory"] is True for answer in self.answers):
             self.add_error(self.MUTUALLY_EXCLUSIVE_CONTAINS_MANDATORY)
 
-        if self.answers[-1]["type"] != "Checkbox":
-            self.add_error(self.NON_CHECKBOX_ANSWER, answer_id=self.answers[-1]["id"])
+        if (
+            self.answers[-1]["type"] != "Checkbox"
+            and self.answers[-1]["type"] != "Radio"
+        ):
```

Comment:
We should use the `AnswerType` enum here and also this should be replaced by an `in` check. so `type not in [checkbox, radio]`